eng/scripts/Get-TypeSpec-Folders.ps1

```diff
@@ -48,13 +48,22 @@ else {
   }
 }
 
+$errTspConfig = @()
```

Comment:
Please move this part of a check to a rule in `tsv`.  We want this rule to fire if a spec author manually runs `tsv` on a folder with `tspconfig.yml`.

In general, we want to put as much validation as possible into `tsv` itself, and only use the pwsh scripts to find potential spec folders.  I'm even considering moving the functionality from the scripts into `tsv` itself, but waiting for more evidence if this is a good idea.